packages/button/button.scss

```
@@ -43,7 +43,7 @@ $hover-elevation-distance: -0.25rem;
     border: none;
     outline: none;
     transform-origin: 50% 90%;
-    transition: transform $animation-timing;
+    transition: transform $animation-timing, background-color 100ms ease-out;
```

Kanskje vi kan benytte anledningen til å bytte til `motion`-mixinen fra `core` her?

```suggestion
    @include motion("standard", "energetic"); // 75ms, bruk evt "productive" for 150ms
    transition-property: transform, background-color;
```

Så kan man fjerne den proprietære `$animation-timing`-variabelen 😊 Pass også på at `mixins/_motion.scss` (evt. `mixins/_all.scss`) er importert hvis du vil ta inn forslaget.

---

Kult, skal sjekke det ut :)